### PR TITLE
Lpc4088

### PR DESCRIPTION
**Pull request by johosemi**
Minimal testing. This is just a copy of lpc1768 with a RAM size change

### CODE REVIEW

File: pyOCD/flash/__init__.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
          'lpc800': Flash_lpc800,
          'lpc11u24': Flash_lpc11u24,
          'lpc1768':  Flash_lpc1768,
+         'lpc4088':  Flash_cortex_m,
```
Comment (c1728p9) [inline pyOCD/flash/__init__.py]: Can you use the lpc1768's flash algorithm?  The one you have selected now will not allow you to program the flash.

File: pyOCD/board/mbed_board.py [hunk 2]
```diff
@@ -35,6 +35,7 @@
                 "9004": "lpc1768",
                 "1040": "lpc11u24",
                 "1050": "lpc800",
+                "1060": "lpc4088",
```
Comment (c1728p9) [inline pyOCD/board/mbed_board.py]: What is the name of this board? The "EA LPC4088 QuickStart Board"?  I'm adding board name in #129

**Comment by 0xc0170** [comment]
@johosemi there are some comments, can you look at them?

**Comment by 0xc0170** [comment]
there has not been any comments for a while. @johosemi I'll close this now. You can reopen it . Thanks for understanding

**Pull request closed without merging**